### ISSUE TITLE
GREENMAIL_SIEVE_IGNORE_DETAIL is always on

### ISSUE THREAD

**Issue by shlomi-lr** [greenmail-mail-test/greenmail#823]
The following condition test will never have a `null` for `sieveIgnoreDetail`:
https://github.com/greenmail-mail-test/greenmail/commit/acaa0175ab7678720c4cbad2d89b188c4032431c#diff-3f3472a9f5089fa5be56cb52816e9316328638b8951fc0132f986fff1cc2f332R78

imo, we really meant this, right? 

```java
String sieveIgnoreDetail = properties.getProperty(GREENMAIL_SIEVE_IGNORE_DETAIL, "false");
        if (Boolean.TRUE.toString().equalsIgnoreCase(sieveIgnoreDetail)) {
            configuration.withSieveIgnoreDetail();
        }
```

**Comment by shlomi-lr** [comment]
[GREENMAIL_SIEVE_IGNORE_DETAIL.patch](https://github.com/user-attachments/files/17922524/GREENMAIL_SIEVE_IGNORE_DETAIL.patch)

**Comment by shlomi-lr** [comment]
see https://github.com/greenmail-mail-test/greenmail/pull/824

**Comment by marcelmay** [comment]
PR was included in 2.1.2
